### PR TITLE
Paris - Bastien & Poeli

### PR DESCRIPTION
**Pull request by pbojork**
We managed to set up the content in the HTML for the most part.
Had issues with Bootstrap, mostly to modifying the components (carousel) and fontawesome icons.

@basttran

### CODE REVIEW

File: starter-code/index.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+          <p>
+            Cras aliquet venenatis porttitor. Donec rutrum sapien et sapien
+            dignissim, eu dapibus mi molestie. Suspendisse dictum convallis quam
+            et sodales. Praesent non enim et magna congue gravida.
+          </p>
+        </div>
+      </div>
+    </header>
+
+    <section>
```
Comment (ta-web-paris) [inline starter-code/index.html]: This bloc is not correct, the responsive doesn't work. I correct it to you : 
```
<section>

<div class="container">

        <div class="row">

          // // Knowing that the bootstrap grid is divided by 12 columns, the section will takes all the width of the screen
 
          <div class="section-tilte col-12">
            <h2>Latest Project</h2>
            <p>FEW OF THE LATEST PORTFOLIO ITEMS</p>
            <hr />
          </div>

          <div class="row flex-photo">

            // Knowing that the bootstrap grid is divided by 12 columns, this means : 
            // * col-lg-3 = on computer screen (lg = large) : each 4 columns will takes 3/12 (total 3x4 = 12), so 25% of the screen. So the 4 blocks will be on the same line
            // * col-md-6 = on tablet screen (md = medium) : each 4 columns will takes 6/12, so 50% of the screen, so there will be one first line with 2 blocs and another line with the last 2 blocs
            // * col-sm-12 = on smartphone screen (sm = small) : each 4 columns will takes 12/12, so 100% of the screen, so there will be one bloc per line
            
            <div class="col-lg-3 col-md-6 col-sm-12 postbox">
              <img src="./img/project-1.jpg" alt="mec-muscu" />
              <p>dfa efweqcfeqw</p>
            </div>
            <div class="col-lg-3 col-md-6 col-sm-12 postbox">
              <img src="./img/project-2.jpg" alt="cat" />
              <p>xdwdwedxwe edfcwedfcw</p>
            </div>
            <div class="col-lg-3 col-md-6 col-sm-12 postbox">
              <img src="./img/project-3.jpg" alt="flour" />
              <p>fwecfweff wefqewfcqwe</p>
            </div>
            <div class="col-lg-3 col-md-6 col-sm-12 postbox">
              <img src="./img/project-4.jpg" alt="girl" />
              <p>fxeqwfewfxwefcw fwe fwec</p>
            </div>
          </div>

        </div>

</section>
``

File: starter-code/index.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        <img src="./img/project-3.jpg" />
+        <h6>rrqlgfknserglkrseg</h6>
+      </article>
+      <article>
+        <img src="./img/project-4.jpg" />
+        <h6>rrqlgfknserglkrseg</h6>
+      </article>
+    </section>
+
+    <section>
```
Comment (ta-web-paris) [inline starter-code/index.html]: Same correction here because same grid :)

**Comment by ta-web-paris** [comment]
I know that you had not much time to do this exercise, but I really encourage you to finish it by yourself this week(-end).
You’ll need to use bootstrap during your projects so that’s better to practice before to not loose time with it after :)
If you’re looking for the correction, here it is : https://github.com/ironhack-dev-squad-107/html-css-bootstrap
Bootstrap grid system : https://getbootstrap.com/docs/4.1/layout/grid/
Good luck :)

**Comment by basttran** [comment]
Will do. Thx!

**Pull request closed without merging**